src/main/java/sjchat/restapi/entities/ChatRequest.java

```diff
@@ -0,0 +1,15 @@
+package sjchat.restapi.entities;
+
+public class ChatRequest {
+
+  private String title;
+  private long[] users;
```

Comment:
These entity classes have been updated by the database branch and thus I am not changing them for now.